src/sql/workbench/contrib/backup/browser/backupActions.ts

```diff
@@ -55,17 +58,20 @@ export class BackupAction extends Task {
 		if (profile) {
 			const serverInfo = connectionManagementService.getServerInfo(profile.id);
 			if (serverInfo && serverInfo.isCloud && profile.providerName === mssqlProviderName) {
-				return accessor.get<INotificationService>(INotificationService).info(localize('backup.commandNotSupported', "Backup command is not supported for Azure SQL databases."));
+				return accessor.get<INotificationService>(INotificationService).info(backupNotSupportedForAzure);
 			}
 
 			if (!profile.databaseName && profile.providerName === mssqlProviderName) {
-				return accessor.get<INotificationService>(INotificationService).info(localize('backup.commandNotSupportedForServer', "Backup command is not supported in Server Context. Please select a Database and try again."));
+				return accessor.get<INotificationService>(INotificationService).info(backupNotSupportedOutOfDBContext);
 			}
 		}
 
 		const capabilitiesService = accessor.get(ICapabilitiesService);
 		const instantiationService = accessor.get(IInstantiationService);
 		profile = profile ? profile : new ConnectionProfile(capabilitiesService, profile);
+		if (!profile.databaseName) {
```

Comment:
please note that this (and same in other file) is the real change - rest is just string cleanup.